### PR TITLE
More use of JS string template literals. NFC

### PR DESCRIPTION
**Pull request by sbc100**
Also, just `dbg` over `out` where appropriate.

### CODE REVIEW

File: src/library_sdl.js [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ var LibrarySDL = {
         audio.webAudioNode['start'](0, audio.currentPosition);
         audio.startTime = SDL.audioContext['currentTime'] - audio.currentPosition;
       } catch(e) {
-        err('playWebAudio failed: ' + e);
+        err(`AplayWebAudio failed: ${e}`);
```
Comment (tlively) [inline src/library_sdl.js:1153]:
```suggestion
        err(`playWebAudio failed: ${e}`);
```

File: src/settings_internal.js [hunk 2]
```diff
@@ -265,3 +265,5 @@ var POST_JS_FILES = [];
 var PTHREADS = false;
 
 var BULK_MEMORY = false;
+
+var MINIFY_JS = true;
```
Comment (tlively) [inline src/settings_internal.js:269]: This looks like a more consequential change.

**Review by tlively — COMMENTED** [review]
Any idea why JS size went up slightly in so many cases? Is that just a rebaselining artifact?

File: src/settings_internal.js (hunk 2 above)
Comment (sbc100) [reply src/settings_internal.js:269]: Sorry, somehow that leaked from another PR I'm working on

**Review by tlively — APPROVED** [review]

**Pull request merged**